docs/pages/User/Install/_index.md

```diff
@@ -1,24 +1,134 @@
+* **ServiceCIDR**:  the address space of IPs assigned to ClusterIP services
+
+{{% notice note %}}
+Liqo only supports Kubernetes >= 1.19.0.
+{{% /notice %}}
+
+Depending on the provider, you have different way to retrieve those parameters. For more information, you can check in the following subsections:
+
+{{%expand "Kubeadm" %}}
+
+To retrieve PodCIDR and ServiceCIDR in a Kubeadm cluster, you can just extract it from the kube-controller-manager spec:
+
+```bash
+POD_CIDR=$(kubectl get pods --namespace kube-system --selector component=kube-controller-manager --output jsonpath="{.items[*].spec.containers[*].command}" 2>/dev/null | grep -Po --max-count=1 "(?<=--cluster-cidr=)[0-9.\/]+")
+SERVICE_CIDR=$(kubectl get pods --namespace kube-system --selector component=kube-controller-manager --output jsonpath="{.items[*].spec.containers[*].command}" 2>/dev/null | grep -Po --max-count=1 "(?<=--service-cluster-ip-range=)[0-9.\/]+")
+echo "POD CIDR: $POD_CIDR"
+echo "SERVICE CIDR: $SERVICE_CIDR"
+```
+{{% /expand%}}
+
+{{%expand "AWS Elastic Kubernetes Service (EKS)" %}}
+Retrieve information about your clusters, typing:
+```bash
+API_SERVER=$(aws eks describe-cluster --name liqo-cluster --region=eu-central-1 | jq -r .cluster.endpoint | sed 's/https:\/\///g' ))
+
+```
+{{% /expand%}}
+
+{{%expand "Azure Kubernetes Service (AKS)" %}}
+
+AKS clusters have by default with the following PodCIDR and ServiceCIDR:
+
+If you are using Azure CNI:
+
+```bash
+SUBNET_ID=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r '.[0].agentPoolProfiles[0].vnetSubnetId')
+POD_CIDR=$(az network vnet subnet show --ids ${SUBNET_ID} | jq -r .addressPrefix)
+SERVICE_CIDR=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r ".[0].networkProfile.serviceCidr")
+```
+
+Or Kubenet:
+
+```bash
+POD_CIDR=$(az network vnet subnet show --ids ${SUBNET_ID} | jq -r ".[0].networkProfile.serviceCidr)
+SERVICE_CIDR=$(az aks list --query="[?name=='__YOUR_CLUSTER_NAME__']" | jq -r ".[0].networkProfile.serviceCidr"
+```
+
+
+{{% /expand%}}
+{{%expand "Google Kubernetes Engine (GKE)" %}}
+
+```bash
+SERVICE_CIDR=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.servicesIpv4Cidr`)
+POD_CIDR=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.clusterIpv4Cidr`)
+API_SERVER=$(gcloud container clusters describe __YOUR_CLUSTER_NAME__ --zone -__YOUR_ZONE__ --project __YOUR_PROJECT_ID__ --format="json" | jq -r `.endpoint`)
+```
+
+{{% /expand%}}
+{{%expand "K3s" %}}
+
+K3s clusters have by default with the following PodCIDR and ServiceCIDR:
+
+| Variable               | Default | Description                                 |
+| ---------------------- | ------- | ------------------------------------------- |
+| `networkManager.config.podCIDR`             |    10.42.0.0/16     |
```

Comment:
Am I wrong or the "description" column here is missing?

Reply:
You are right, ty